drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/menu/ScenarioContextMenuRegistry.java

```diff
@@ -255,4 +257,14 @@ private boolean manageHeaderRightClick(final ScenarioGrid scenarioGrid,
         scenarioGrid.setSelectedColumnAndHeader(uiHeaderRowIndex, uiColumnIndex);
         return true;
     }
+
+
+    public void setErrorReportPopupPresenter(ErrorReportPopoverPresenter errorReportPopupPresenter) {
+        this.errorReportPopupPresenter = errorReportPopupPresenter;
+    }
+
+    public void hideErrorReportPopupover() {
```

Comment:
Just double checking naming, is `hideErrorReportPopupover` correct, the `over` suffix seems to be not needed to me.

Reply:
The correct name is hideErrorReportPopover.
DONE